### PR TITLE
[Train] Remove custom labels from Llama-2 compute config

### PR DESCRIPTION
**Pull request by kouroshHakha**
## Why are these changes needed?

It is often hard to explain why we need large_cpu_mem = 0.01 in the custom resources, and also I have found that g5.16xlarge nodes are not that available anymore. So launching this template may get blocked because of the in-availability of the resources. 

This PR makes a simplification by separating 7B and 13B compute configs. This allows me to specificy 16xg5.4xlarge nodes for 7B so that I can quickly start with something. It also allows me to use g5.12xlarge nodes for 13B (subject to availability).

Since g5.12xlarge can be tricky to get, I removed the release test for 13B models. 

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/templates/04_finetuning_llms_with_deepspeed/compute_configs/aws_7b_or_13b.yaml
Comment (kouroshHakha) [inline doc/source/templates/04_finetuning_llms_with_deepspeed/compute_configs/aws_7b_or_13b.yaml:1]: I have not update the name of this file to aws_7b.yaml, because it will break the current system unless I make a separate PR to the product and let that get deployed. I don't want to be blocked on the deployment of the product.

**Comment by kouroshHakha** [comment]
Release test: https://buildkite.com/ray-project/release-tests-pr/builds/49975

File: doc/source/templates/04_finetuning_llms_with_deepspeed/README.md [hunk 1]
```diff
@@ -23,7 +23,13 @@ And launch the following script:
 
 The flag `--as-test` is for demo / testing purposes as it runs through only one forward and backward pass of the model. The model loading, and remote checkpointing would still run. 
 
-Similarly for 13B you can do
+Similarly for 13B you need a different compute config. 
+
+|            | num | instance type | GPU per node | GPU Memory | CPU Memory |
+|------------|-----|---------------|--------------|------------|------------|
+| Head node  | 1   | m5.xlarge   | -     | -     | -     |
+| Worker node| 4  | g5.12xlarge    | 4 x A10G     | 24 GB      | 64 GB      |
```
Comment (justinvyu) [inline doc/source/templates/04_finetuning_llms_with_deepspeed/README.md:31]: nice

File: doc/source/templates/04_finetuning_llms_with_deepspeed/compute_configs/aws_7b_or_13b.yaml [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
+# Autoscale to 16 g5.4xlarge --> 16 A10Gs
 region: us-west1
 allowed_azs: [any]
 head_node_type:
     name: head_node
-    instance_type: g5.16xlarge
-    resources:
-      custom_resources: 
-        large_cpu_mem: 1
+    instance_type: m5.xlarge
```
Comment (justinvyu) [inline doc/source/templates/04_finetuning_llms_with_deepspeed/compute_configs/aws_7b_or_13b.yaml:6]: Can we bump the size of the headnode. 4 CPUs is a bit small. the driver still does some nontrivial stuff (writing to a bunch of files, etc.)

**Comment by kouroshHakha** [comment]
tests passed.

**Review by justinvyu — APPROVED** [review]
